### ISSUE TITLE
remove Peter Uhe (pfu599) from accessdev administrator

### ISSUE THREAD

**Issue by penguian** [ACCESS-NRI/accessdev-Trac-archive#152]
`resolution_fixed`   |    by axs599@nci.org.au

___

Need to update puppet/hieradata/project.yaml

Are there any implication elsewhere?

(I am just trying out accessdev admin with Say and using this as a example item for consideration) 

___
_Issue migrated from [trac:152](https://accessdev.nci.org.au/trac/152) at 2024-01-31 18:08:53 +1100_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed status from `new` to `closed`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _changed resolution from `` to `fixed`_

**Comment by penguian** [comment]
_@scott.wales@bom.gov.au_ _commented_


___
Fixed in 4edd9c4

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _changed owner from `` to `Scott Wales <scott.wales@unimelb.edu.au>`_

**Comment by penguian** [comment]
_Scott Wales <scott.wales@unimelb.edu.au>_ _committed [4]_
___
In [changeset:"4edd9c452bfe2ddb5e6b400a2791ecd702f682f9/puppet"]:
```
#CommitTicketReference repository="puppet" revision="4edd9c452bfe2ddb5e6b400a2791ecd702f682f9"
Remove Peter Uhe from admins

Thanks for your help Peter

Fixes #152
```
